### DISCUSSION TITLE
`svelte/no-at-html-tags`, so what?

### DISCUSSION THREAD

**Discussion by babakfp** [sveltejs/eslint-plugin-svelte#391] [Q&A] [answered]
![Screenshot 2023-03-03 225116](https://user-images.githubusercontent.com/44144724/222880784-c934020b-80d1-4898-91cf-f1348d881c12.png)

What do you want me to do with this? I don't want to look at it every single time that I run eslint. Should I just disable it?

**Comment by ota-meshi** [comment, marked as answer]
You are free to turn off the rule.
